env/archivist

```diff
@@ -17,4 +18,8 @@ WS_MAX_CONNECTIONS=10000
 RPC_PORT_MAP="${RPC_PORT}:${RPC_PORT}"
 WS_PORT_MAP="${WS_PORT}:${WS_PORT}"
 PORT_MAP="${PORT}:${PORT}"
+VALIDATOR_PORT_MAP="${VALIDATOR_PORT}:${VALIDATOR_PORT}"
```

Comment:
I'm concerned this (similarly PORT) is even required for archivist (=non-validator in my mind). Are archivists run without the `--validator` flag? I guess they should.

Reply:
You're right -- fixed :)